6-boy-scout-rule/README.md

```diff
@@ -0,0 +1,10 @@
+###Boy Scout Rule
+    Always leave the code you're editing better than you found it. 
+    ~ Robert C. Martin (Uncle Bob)
+
+![](../images/boy-scout-rule.jpg)
+
+### Speaker notes
```

Comment:
IMO better you create a separate doc for Speaker notes, so as we discussed we use the second monitor to refer to.  Come to think about it I should also create some.

Can you share a doc amongst ourselves?

But do keep some abstract here as well so the audience would see it

Reply:
let's discuss about this tomorrow and move to a better place